File: bot/pages/more.py
from telegram import InlineKeyboardMarkup, InlineKeyboardButton
from telegram.ext import ContextTypes

def create_message(context: ContextTypes.DEFAULT_TYPE) -> dict:
    buttons = [[
        InlineKeyboardButton(text=context._chat_data.get_lang()['button.calls'], callback_data='open.calls'),
        InlineKeyboardButton(text=context._chat_data.get_lang()['button.left'], callback_data='open.left')
    ], [
        InlineKeyboardButton(text=context._chat_data.get_lang()['button.info'], callback_data='open.info')
    ], [
        InlineKeyboardButton(text=context._chat_data.get_lang()['button.back'], callback_data='open.menu')
    ]]

    return {
        'text': context._chat_data.get_lang()['page.more'],
        'reply_markup': InlineKeyboardMarkup(buttons),
        'parse_mode': 'MarkdownV2'
    }

File: internal/dteubot/pages/more.go
/*
 * Copyright (c) 2022 Bohdan Marukhnenko
 *
 * Permission is hereby granted, free of charge, to any person obtaining a copy
 * of this software and associated documentation files (the "Software"), to deal
 * in the Software without restriction, including without limitation the rights
 * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
 * copies of the Software, and to permit persons to whom the Software is
 * furnished to do so, subject to the following conditions:
 *
 * The above copyright notice and this permission notice shall be included in all
 * copies or substantial portions of the Software.
 *
 * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
 * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
 * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
 * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 * SOFTWARE.
 */

package pages

import (
	"github.com/PaulSonOfLars/gotgbot/v2"
	"github.com/cubicbyte/dteubot/internal/i18n"
)

func CreateMorePage(lang i18n.Language) (Page, error) {
	page := Page{
		Text: lang.Page.More,
		ReplyMarkup: gotgbot.InlineKeyboardMarkup{
			InlineKeyboard: [][]gotgbot.InlineKeyboardButton{
				{{
					Text:         lang.Button.CallsSchedule,
					CallbackData: "open.calls",
				}, {
					Text:         lang.Button.TimeLeft,
					CallbackData: "open.left",
				}},
				{{
					Text:         lang.Button.StudentsList,
					CallbackData: "open.students_list",
				}},
				{{
					Text:         lang.Button.Info,
					CallbackData: "open.info",
				}},
				{{
					Text:         lang.Button.Back,
					CallbackData: "open.menu",
				}},
			},
		},
		ParseMode: "MarkdownV2",
	}

	return page, nil
}
